spinta/backends/__init__.py

```diff
@@ -23,6 +23,25 @@ def prepare(context: Context, type: Type, backend: Backend, value: object) -> ob
     return value
 
 
+@prepare.register()
+def prepare(context: Context, type: Array, backend: Backend, value: object) -> object:
+    # prepare array and it's items for datastore
+    array_item_type = type.items.type
+    new_prepare_array = []
+    for item in value:
+        new_prepare_array.append(prepare(context, array_item_type, backend, item))
+    return new_prepare_array
```

Comment:
It is always more difficult to read too verbose code with lots of unnecessary operations. You can do the same with this:

```python
return [prepare(context, type.items.type, backend, v) for v in value]
```

At least to my eye, it is very clear, that each item of `value` is mapped to `prepare` function call.

In you case, I see new variables introduced, `array_item_type` is used just once, also there are three operations with `new_prepare_array`. It just takes more time to undrestand.